projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -72,22 +86,45 @@ export default function SubscribePanels() {
 	// We still need to render the access level selector, as historical posts need a way to edit their access level for people visiting them on the web.
 	// TODO: Additionally, pages also can be protected. They will not send an email, but can be a resource that needs the acces selector.
 	if ( postWasEverPublished ) {
-		return <AccessLevelSelectorPanel setPostMeta={ setPostMeta } accessLevel={ accessLevel } />;
+		return (
+			<AccessLevelSelectorPanel
+				setPostMeta={ setPostMeta }
+				accessLevel={ accessLevel }
+				followerCount={ followerCount }
+				subscribersCount={ subscribersCount }
+				paidSubscribersCount={ paidSubscribersCount }
+			/>
+		);
 	}
 
 	// Do not show any panels when we have no info about the subscriber count, or it is too low.
 	if (
-		( ! Number.isFinite( subscriberCount ) || subscriberCount <= 0 ) &&
+		( ! Number.isFinite( subscribersCount ) || subscribersCount <= 0 ) &&
+		( ! Number.isFinite( paidSubscribersCount ) || paidSubscribersCount <= 0 ) &&
 		( ! Number.isFinite( followerCount ) || followerCount <= 0 )
 	) {
 		return null;
 	}
 
-	const showNotices = Number.isFinite( subscriberCount ) && subscriberCount > 0;
+	// In the paid-subscriber only, we send to a restricted number of subscribers
+	const realSubscribersCount =
+		accessOptions[ accessLevel ] === accessOptions.paid_subscribers
+			? paidSubscribersCount
+			: subscribersCount;
+	// We send to the social followers only when it is "everybody"
+	const realFollowersCount =
```

Comment:
It seems to me that the `subscriberCount ` was a good name and this logic should live in the 

```
useEffect( () => {
		getSubscriberCounts( counts => {
```
block?